create-register.sh

```diff
@@ -0,0 +1,257 @@
+  cd $CURRENT_WORKING_DIR
+
+  case $ENV in
+      preview)
+          RDS_MINT_END_POINT=preview-mint.ckm2jre98w8b.eu-west-1.rds.amazonaws.com
+          RDS_PRESENTATION_END_POINT=preview-presentation.ckm2jre98w8b.eu-west-1.rds.amazonaws.com
+          ;;
+      prod)
+          RDS_MINT_END_POINT=prod-mint.ckm2jre98w8b.eu-west-1.rds.amazonaws.com
+          RDS_PRESENTATION_END_POINT=prod-presentation.ckm2jre98w8b.eu-west-1.rds.amazonaws.com
+          ;;
+      *)
+          echo "Unrecognized environment: $ENV"
+          usage; exit 1
+          ;;
+  esac
+
+  MINT_USER_PASSWORD=`pass ${ENV}/rds/mint/${REGISTER_DB_NAME}`
+  READ_ONLY_MINT_USER_PASSWORD=`pass ${ENV}/rds/mint/${REGISTER_DB_NAME}_readonly`
+  PRESENTATION_USER_PASSWORD=`pass ${ENV}/rds/presentation/${REGISTER_DB_NAME}`
+  READ_ONLY_PRESENTATION_USER_PASSWORD=`pass ${ENV}/rds/presentation/${REGISTER_DB_NAME}_readonly`
+
+
+  REGISTER_CONFIG_DIRECTORY=$CONFIG_FILE_DIRECTORY/$REGISTER_NAME
+
+  MINT_CONFIG_DIRECTORY=$REGISTER_CONFIG_DIRECTORY/mint
+
+  PRESENTATION_CONFIG_DIRECTORY=$REGISTER_CONFIG_DIRECTORY/presentation
+
+  mkdir -p $CONFIG_FILE_DIRECTORY $REGISTER_CONFIG_DIRECTORY $MINT_CONFIG_DIRECTORY $PRESENTATION_CONFIG_DIRECTORY
+
+
+echo "database:
+  driverClass: org.postgresql.Driver
+  url: jdbc:postgresql://$RDS_MINT_END_POINT:5432/$REGISTER_DB_NAME
+  user: $REGISTER_DB_NAME
+  password: $MINT_USER_PASSWORD
+
+  #db connection properties
+  initialSize: 1
+  minSize: 1
+  maxSize: 4
+
+  properties:
+    charSet: UTF-8
+
+server:
+  applicationConnectors:
+  - type: http
+    port: 4567
+" > $MINT_CONFIG_DIRECTORY/mint-config.yaml
+
+
+
+echo "server:
+  registerDefaultExceptionMappers: false
+
+database:
+  driverClass: org.postgresql.Driver
+  url: jdbc:postgresql://$RDS_PRESENTATION_END_POINT:5432/$REGISTER_DB_NAME
+  user: ${REGISTER_DB_NAME}_readonly
+  password: ${READ_ONLY_PRESENTATION_USER_PASSWORD}
+
+  #db connection properties
+  initialSize: 1
+  minSize: 1
+  maxSize: 4
+
+  properties:
+    charSet: UTF-8
+" > $PRESENTATION_CONFIG_DIRECTORY/config.yaml
+
+  #Removing indexer/properties first so that we always use the file downloaded from s3
+  rm -f $INDEXER_CONFIG_LOCATION
+
+  aws s3 cp s3://$BUCKET_NAME/indexer/indexer.properties $INDEXER_CONFIG_LOCATION
+
+  #Confirm that the file is successfuly downloaded from s3
+  if [ ! -f $INDEXER_CONFIG_LOCATION ]; then
+    exit 1
+  fi
+
+  echo "$REGISTER_NAME.source.postgres.db.connectionString=jdbc:postgresql://$RDS_MINT_END_POINT:5432/${REGISTER_DB_NAME}?user=${REGISTER_DB_NAME}_readonly&password=${READ_ONLY_MINT_USER_PASSWORD}" >> $INDEXER_CONFIG_LOCATION
+  echo "$REGISTER_NAME.destination.postgres.db.connectionString=jdbc:postgresql://$RDS_PRESENTATION_END_POINT:5432/${REGISTER_DB_NAME}?user=${REGISTER_DB_NAME}&password=${PRESENTATION_USER_PASSWORD}" >> $INDEXER_CONFIG_LOCATION
+}
+
+function loadConfigurationToS3(){
+  cd $CURRENT_WORKING_DIR
+  aws s3 sync $CONFIG_FILE_DIRECTORY s3://$BUCKET_NAME
```

Comment:
will this download the whole bucket? we should probably just copy what we need up rather than leaving unencrypted passwords on local machine